include/amber/amber.h

```diff
@@ -57,6 +57,8 @@ struct Options {
   EngineType engine;
   /// Holds engine specific configuration. Ownership stays with the caller.
   EngineConfig* config;
+  /// The SPIR-V environment to target.
+  uint32_t spv_env;
```

Comment:
We use uint32_t throughout instead of spv_target_env because amber can be compiled without spirv-tools. So, we can't use the spirv-tools env. We wait until the last minute to cast the uint32_t to the spv_target_env.

Reply:
You could store the environment string and defer parsing until compiling the shader where you already have the SPIRV-Tools requirement. You might parse it once per shader though in that case.

Reply:
That's nicer, done. Also rolled DEPS to pickup new spirv-tools and spirv-headers.